components/kyma-environment-broker/internal/process/update/initialisation_step.go

```diff
@@ -28,19 +29,29 @@ const (
 
 const postUpgradeDescription = "Performing post-upgrade tasks"
 
+//go:generate mockery --name=RuntimeVersionConfiguratorForUpdating --output=automock --outpkg=automock --case=underscore
+type RuntimeVersionConfiguratorForUpdating interface {
+	ForUpdating(op internal.UpdatingOperation) (*internal.RuntimeVersionData, error)
+}
+
 type InitialisationStep struct {
-	operationManager *process.UpdateOperationManager
-	operationStorage storage.Operations
-	instanceStorage  storage.Instances
-	inputBuilder     input.CreatorForPlan
+	operationManager       *process.UpdateOperationManager
+	operationStorage       storage.Operations
+	instanceStorage        storage.Instances
+	runtimeStatesStorage   storage.RuntimeStates
+	runtimeVerConfigurator RuntimeVersionConfiguratorForUpdating
```

Comment:
runtimeStateStorage and runtimeVerConfigurator - where is it used?

Reply:
nowhere, this is a leftover from the one of options I was trying to implement for fetching last runtimeID. Thanks for spotting it, will be removed